topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTopicDeletionDisabledIT.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.topic;
+
+import io.debezium.kafka.KafkaCluster;
+import io.strimzi.api.kafka.Crds;
+import io.strimzi.api.kafka.model.KafkaTopic;
+import io.strimzi.test.BaseITST;
+import io.vertx.core.Future;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.nio.file.Files;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+@RunWith(VertxUnitRunner.class)
+public class TopicOperatorTopicDeletionDisabledIT extends TopicOperatorBaseIT {
+
+    private static final Logger LOGGER = LogManager.getLogger(TopicOperatorTopicDeletionDisabledIT.class);
+
+    @Before
+    public void setup(TestContext context) throws Exception {
```

Comment:
I suspect we could extract a method for starting the kafka cluster from this which we put in the base class, but I'm happy for that to be done in an other PR if the need to merge this is pressing.